tests/system/providers/amazon/aws/example_databrew.py

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import pendulum
+
+from airflow.decorators import dag
+from airflow.models.baseoperator import chain
+from airflow.providers.amazon.aws.operators.glue import (
+    GlueDataBrewStartJobOperator,
+)
+from tests.system.providers.amazon.aws.utils import SystemTestContextBuilder
+
+DAG_ID = "example_databrew_dag"
+
+# Externally fetched variables:
+JOB_NAME = "JOB_NAME"
+
+
+sys_test_context_task = SystemTestContextBuilder().add_variable(JOB_NAME).build()
```

Comment:
We can generate the job name ourselves

```suggestion
sys_test_context_task = SystemTestContextBuilder().build()
```